non-standard.json

```diff
@@ -14,6 +14,7 @@
   "org.scala-lang scalap":                           "version",
   "amplab spark-indexedrdd":                         "pom",
   "me.leaf akka-persistence-android":                "pom",
+  "com.typesafe config":                             "pom",
```

Comment:
Hm... so this won't work you mean?
The solution does not seem that hard... maybe I can try to implement it - thanks for pointing to the right point in the source.